nixos/modules/services/web-apps/isso.nix

```diff
@@ -0,0 +1,56 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.isso;
+
+  inherit (lib) mkEnableOption mkIf mkOption types;
+in {
+
+  options = {
+    services.isso = {
+      enable = mkEnableOption "Whether to enable isso, a commenting server similar to Disqus.";
+
+      user = mkOption {
+        type = types.str;
+        default = "isso";
+        description = "isso service user.";
+      };
+
+      group = mkOption {
+        type = types.str;
+        default = "isso";
+        description = "isso service group.";
+      };
+
+      config = mkOption {
+        description = "Isso configuration file content.";
+        type = types.lines;
+        default = "";
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    environment.etc."isso.conf".text = ''
```

Comment:
It's better to pass `-c ${cfgfile}` rather than enforcing `/etc/isso.conf` IMHO.